angular-workspace/projects/example-client-app/src/app/customapp/customapp.component.html

```diff
@@ -150,6 +150,10 @@
                 <nimble-list-option *ngFor="let item of comboboxItems" [ngValue]="item">{{ item ? item.first : '' }}</nimble-list-option>
             </nimble-combobox>
         </div>
+        <div class="sub-container">
+            <div class="container-label">Spinner</div>
+            <nimble-spinner aria-label="Loading example content"></nimble-spinner>
```

Comment:
I looked at the accessibility tree for the spinner in Firefox and Edge. In Edge it looked sane but in Firefox it showed a message saying "Does not meet the WCAG standards for text alternative. Form elements should have a visible text label" which linked to [this guideline](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names#form_elements_should_have_a_visible_text_label).

Can you do some digging to see if this is a valid warning?

Reply:
Info from Storybook, in case it helps.
https://dequeuniversity.com/rules/axe/4.5/aria-progressbar-name?application=axeAPI

![image](https://user-images.githubusercontent.com/109235103/208172531-98a49808-54cf-4663-95bc-886a1ac7a2a1.png)

Reply:
I think the message is valid.
[Here's another page saying something similar](https://dequeuniversity.com/rules/axe/4.3/label-title-only): "The label is applied directly to the element, and is completely invisible, which is a huge disadvantage for all sighted users, but screen readers will read the label."

Since `role=progressbar` is set by us directly on the `nimble-spinner`, users can set ARIA attributes as needed on the spinner (and we don't need to do work to reflect them onto something in the shadow DOM, like we have to for some other controls).

That message is just saying that the label isn't visible. The spinner could be used with a visible label this way, then the Firefox warning isn't there, and the spinner shows in the accessibility tree with the label's text:
```html
<nimble-spinner id="spinnerId" aria-labelledby="spinnerLabel"></nimble-spinner>
<label id="spinnerLabel">Loading example content...</label>
```

So I think it's fine as-is.